livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -212,13 +232,16 @@ def chat(
             if fnc_ctx and parallel_tool_calls is not None:
                 opts["parallel_tool_calls"] = parallel_tool_calls
 
+        user = self._opts.user or openai.NOT_GIVEN
```

Comment:
is openai.NOT_GIVEN exported? last time I checked it wasn't

Reply:
https://github.com/openai/openai-python/blob/3bf03f2f7ee1a9e214484524000774532d02cba7/src/openai/__init__.py#L45